students/rmart300/trigram/trigram.py

```diff
@@ -0,0 +1,45 @@
+
+trigram_dict={}
+
+def read_book(book_file):
+
+    book = open(book_file, 'r'):
```

Comment:
always make sure to close the file.. and instead of having to worry about that use `with` 🙂